packages/d3fc-webgl/src/buffers/baseAttributeBuilder.js

```diff
@@ -7,13 +7,13 @@ export default () => {
     let normalized = false;
     let stride = 0;
     let offset = 0;
-    let validSize = 0;
+    let hasDataBeenRendered = false;
```

Comment:
I'm not sure this name is completely accurate, maybe `hasPropertyChanged` or `hasAttributeChanged` could be more descriptive

Reply:
`dirty`?

Reply:
I've changed to `hasPropertyChanged` (thus inverting the boolean).